addons/hr_timesheet/views/project_task_views.xml

```diff
@@ -24,7 +24,7 @@
                             (incl. <field name="subtask_allocated_hours" nolabel="1" widget="timesheet_uom_no_toggle" class="oe_inline"/> on
                             <span class="fw-bold text-dark"> Sub-tasks</span>)
                         </span>
-                        <span invisible="not project_id">(<field name="progress" class="oe_inline" nolabel="1" widget="integer"/> %)</span>
+                        <field name="progress" invisible="not project_id" class="oe_inline" nolabel="1" widget="project_task_progress" decoration-danger="progress >= 101"/>
```

Comment:
```suggestion
                        <field name="progress" invisible="not project_id" class="oe_inline" nolabel="1" widget="project_task_progress" decoration-danger="progress > 100"/>
```
makes more sense to me

Reply:
actually, we're using integer widget for progress so if it's 100.65%, it will be displayed 100 and if it's 100, it will also be displayed 100, they're both 100 but one in red and the second one not,
i suggested removing the integer widget but AJU preferred that hack of >= 101

Reply:
my bad, i though it's rounded down